contracts/StrategyCurveGeist.sol

```diff
@@ -130,7 +110,23 @@ abstract contract StrategyCurveBase is BaseStrategy {
         return balanceOfWant();
     }
 
+    function _claimRewards() internal {
+        // Claims any pending CRV
+        //
+        // Mints claimable CRV from the factory gauge. Reward tokens are sent to `msg.sender`
+        gaugeFactory.mint(address(gauge));
+
+        // harvest third-party rewards from the gauge, if any
+        gauge.claim_rewards();
```

Comment:
this doesn't revert if there aren't rewards, right?

Reply:
Correct